streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBStoreTest.java

```diff
@@ -338,22 +333,42 @@ public void setConfig(final String storeName, final Options options, final Map<S
         return entries;
     }
 
-    private static class ConfigurableProcessorContext extends MockProcessorContext {
-        final Map<String, Object> configs;
-
-        ConfigurableProcessorContext(final File stateDir,
-                                     final Serde<?> keySerde,
-                                     final Serde<?> valSerde,
-                                     final RecordCollector collector,
-                                     final ThreadCache cache,
-                                     final Map<String, Object> configs) {
-            super(stateDir, keySerde, valSerde, collector, cache);
-            this.configs = configs;
+    private static class ConfigurableProcessorContext extends AbstractProcessorContext {
+        private final File stateDir;
+
+        ConfigurableProcessorContext(StreamsConfig config, File stateDir) {
```

Comment:
Nit: add `final` to both parameters -- please follow a "one parameter per line" formatting.